src/elements/content-sidebar/activity-feed/annotations/__tests__/AnnotationActivity.test.js

```diff
@@ -95,6 +95,23 @@ describe('elements/content-sidebar/ActivityFeed/annotations/AnnotationActivity',
         },
     );
 
+    test('should render CommentForm if user clicks on the Modify menu item', () => {
+        const activity = {
+            item: {
+                ...mockAnnotation,
+                isPending: false,
+                permissions: { can_edit: true },
+            },
+        };
+
+        const wrapper = mount(<AnnotationActivity {...mockActivity} {...activity} />);
+
+        wrapper.find('AnnotationActivityMenu').simulate('click');
+        wrapper.find('MenuItem').simulate('click');
+
+        expect(wrapper.exists('CommentForm')).toBe(true);
```

Comment:
Could we do a cancel button click and verify that the `CommentForm` doesn't exist as well?